database/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "explorer-database"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0.51"
+base64 = "0.11"
+bigdecimal = "=0.1.0"
+diesel = { version = "1.4.7", features = ["postgres", "numeric", "serde_json"] }
+# Using hacky diesel-derive-enum https://github.com/adwhit/diesel-derive-enum/issues/52
+diesel-derive-enum = { git = "https://github.com/khorolets/diesel-derive-enum.git", branch = "lookup-hack", features = ["postgres"] }
+cached = "0.23.0"
+futures = "0.3.5"
+hex = "0.4"
+num-traits = "0.2.11"
+serde = { version = "1", features = ["derive"] }
+serde_json = "1.0.55"
+tracing = "0.1.13"
+tokio = { version = "1.1", features = ["sync", "time"] }
+
+actix-diesel = { git = "https://github.com/frol/actix-diesel", rev = "3a001986c89dfabfc3c448d8bae28525101b4992" }
+near-lake-framework = "0.5.2"
```

Comment:
Just a thought. If I remember correctly this `database` crate needs only primitives, perhaps it should be enough to use `near-indexer-primitives` crate instead of the entire Lake Framework. https://crates.io/crates/near-indexer-primitives
Just worth trying

Reply:
Good catch! I've updated to just use `near-indexer-primitives` :)